src/pg_partman_bgw.c

```diff
@@ -425,7 +425,7 @@ void pg_partman_bgw_run_maint(Datum arg) {
             , dbname);
 
     resetStringInfo(&buf);
-    appendStringInfo(&buf, "SELECT n.nspname FROM pg_catalog.pg_extension e JOIN pg_catalog.pg_namespace n ON e.extnamespace = n.oid WHERE extname = 'pg_partman'");
+    appendStringInfo(&buf, "SELECT pg_catalog.quote_ident(nspname) FROM pg_catalog.pg_extension e JOIN pg_catalog.pg_namespace n ON e.extnamespace = n.oid WHERE extname = 'pg_partman'");
```

Comment:
That they're not being used is a lack of consistency on my part. As you can tell from the cleanup you've been doing, it's not being the greatest.

Aliases have nothing to do with the schema. It's to avoid ambiguity amongst column names being similar when joining tables. None are now, but it avoids any possibility of it ever being an issue in the future.